src/components/atoms/pageTitle/PageTitle.tsx

```diff
@@ -1,12 +1,23 @@
-import { Typography } from '@mui/material'
+import { Container, SxProps, Typography } from '@mui/material'
 import { Genre, Role } from '../../../common/constants/enums'
-import { convertFromEnumToText } from '../../../common/utils/utils'
+import { enumMappings } from '../../../common/constants/constants'
 
 type PageTitleProps = {
   title: Genre | Role
+  subheaderText?: string
+  sx?: SxProps
 }
 
-export function PageTitle({ title }: PageTitleProps) {
-  const pageTitle = convertFromEnumToText(title)
-  return <Typography variant="h2">{pageTitle}</Typography>
+export function PageTitle({ title, subheaderText, sx }: PageTitleProps) {
+  const pageTitle = enumMappings[title].text
+  return (
+    <Container sx={{ ...sx }} disableGutters={true}>
+      <Typography variant="h1" sx={{ padding: '0px 0px 8px' }}>
```

Comment:
Are we changing the h1 in a different PR to be Kurt's name in the header? We wont want this to be h1 then.

Reply:
Name in header will be its own style or image according to designer. So we'll use h1 for the page headings, I think